test/actions/recover.js

```diff
@@ -213,7 +213,7 @@ test.serial('applyRecoveryWorkflowToGranule dispatches applyWorkflow', async (t)
   nock('https://example.com')
     .get(`/granules/${granuleId}`)
     .reply(200, getGranuleResponse)
-    .put(`/granules/${granuleId}`)
+    .patch(`/granules/${granuleId}`)
```

Comment:
Huh. I would expect more unit test changes. I guess we only explicitly test `applyWorkflowToGranule`? Not necessarily something to be addressed here but it seems like we should be testing the other requests.